cmd/cmd.go

```diff
@@ -42,10 +44,13 @@ func Execute() {
 
 func init() {
 	// Initialise logging
-	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
+	ctrl.SetLogger(zap.New())
```

Comment:
I discovered a way to expose the zap logging flags which allows user to choose between structured logs or human readable for debugging: https://github.com/cert-manager/sample-external-issuer/commit/c8288f7788b5418b8897da4bba8c227d03e8e431 might be worth considering for a future PR.